lib/pixify.js

```diff
@@ -118,8 +128,8 @@ module.exports = function(options, callback) {
                 }))
                 .pipe(gulpif(options.compress, uglify()))
                 .pipe(header(license, {
-                    date: (new Date()).toString(),
-                    name: packageInfo.name,
+                    date: (new Date()).toUTCString(),
+                    name: options.name,
```

Comment:
Overall this is solid work for the license and UTC time. This is the only change I'm not onboard with. Would like to differentiate the browserify standalone name (option.name) with the actual module name. This is in fact used in Pixi. Would like to keep as package name.